TST: distutils Version classes are deprecated (pytest-remotedata)

```
___________ ERROR at setup of test_get_coord_sys_name[hdr4-galactic] ___________

item = <Function test_get_coord_sys_name[hdr4-galactic]>

    def pytest_runtest_setup(item):
    
>       if LooseVersion(pytest.__version__) < LooseVersion("3.6"):

.../pytest_remotedata/plugin.py:65: 
...
E       DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
```

--- a/setup.cfg
+++ b/setup.cfg
@@ -8,6 +8,7 @@ filterwarnings =
     error
     ignore:numpy.ndarray size changed:RuntimeWarning
     ignore:numpy.ufunc size changed:RuntimeWarning
+    ignore:distutils Version classes are deprecated:DeprecationWarning
 
 [bdist_wheel]
 universal = 1

